Moved some extra files related to research to research directory

hgibson1 opened the pull request.

naved001 commented on docs/source/Archives-page.md:
```diff
@@ -26,6 +26,7 @@
 * [SPL Demo](archives-page/SPL-Demo.html)
 * [Vlan Head Nodes](archives-page/Vlan-Head-Nodes.html)
 * [Vlan settings on the switch](archives-page/Vlan-settings-on-the-switch.html)
+* [Big Data at MIT](archives-page/BigDataAtMIT-MRI.html)
```
the title says you are moving things to the research directory, but it looks like you are putting it in the archives directory

hgibson1 commented:
Big Data at MIT seemed out of date so I moved it to archives. ChRIS is MOC research so I added it to docs/source/research and added link in docs/source/research/Radiology.md (which is information about ChRIS)
